src/chain/operation.cpp

```diff
@@ -194,7 +194,7 @@ std::string operation::to_string() const
     if (data.empty())
         ss << opcode_to_string(code);
     else
-        ss << "[ " << encode_base16(data) << " ]";
+        ss << "[" << encode_base16(data) << "]";
 
```

Comment:
This change would require a corresponding change to the from string implementation (to support both formats). It would also constitute a breaking change in the case where API users depended on the format, although we can accept breaking changes in master (3.0.0).